src/super_gradients/training/datasets/detection_datasets/yolo_format_detection.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+import imagesize
+import numpy as np
+from typing import List, Optional
+
+from super_gradients.common.abstractions.abstract_logger import get_logger
+from super_gradients.training.datasets.detection_datasets.detection_dataset import DetectionDataset
+from super_gradients.training.exceptions.dataset_exceptions import DatasetValidationException
+
+from super_gradients.training.datasets.data_formats import ConcatenatedTensorFormatConverter
+from super_gradients.training.datasets.data_formats.default_formats import XYXY_LABEL, LABEL_NORMALIZED_CXCYWH
+
+logger = get_logger(__name__)
+
+
+class YoloV5FormattedDetectionDataset(DetectionDataset):
```

Comment:
fix name

Reply:
Changed in 773b92834d6cdd9d3ec530e5da7069527b171aaa